.claude/commands/open-pr.md

```
@@ -0,0 +1,146 @@
+   ```
+   git push origin HEAD
+   ```
+   If the branch doesn't exist on remote yet, use `--set-upstream`.
+
+---
+
+### STEP 3 — Open the PR
+
+Create the PR targeting `TARGET_BRANCH`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)